packages/fxa-auth-server/lib/senders/emails/templates/subscriptionPaymentExpired/en.ftl

```diff
@@ -1,7 +1,7 @@
 # Variables:
 #  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
-subscriptionPaymentExpired-subject = Credit card for { $productName } expiring soon
-subscriptionPaymentExpired-title = Your credit card is about to expire
+subscriptionPaymentExpired-subject = Credit card for { $productName } expired or expiring soon
```

Comment:
New ftl ids for the `-subject`, `-title`, and `-content` will be needed for the updated copy. 😅

Reply:
ah is this why i am seeing test failures? because the id's are already used and have the old copy?

Reply:
id's updated

Reply:
Per our conversation, you know what to do :) 

For reference (and in case others want to update email copy), we have a [how-to guide](https://mozilla.github.io/ecosystem-platform/reference/emails#change-an-existing-template) documented in Ecosystem Platform, but -  tldr; each ftl id is unique to its copy/string.